tdp/cli/commands/run.py

```diff
@@ -62,13 +62,15 @@ def run(
         raise click.BadParameter(f"{vars} does not exist")
     dag = Dag(collection_path)
 
-    operation = dag.operations.get(node, None)
+    operation = dag.collections.dag_operations.get(
```

Comment:
`Collections` object have a property `operations`, did you miss it ?

Reply:
Yes